Documentation/docs/learn/faq.md

```diff
@@ -0,0 +1,611 @@
+# Frequenty asked questions
+
+## General information and availability
+
+### Do I need VTK to build ITK ?
+
+With the default configuration, ITK does not require VTK. However, some
+optional modules (e.g. [ITKVTKGlue](https://github.com/InsightSoftwareConsortium/ITKVtkGlue)) do require VTK.
+
+ITK does not provide any visualization functionalities. Applications
+requiring visualization should implement those functionalities from
+internal code or libraries like OpenGL, GLUT or VTK.
+
+### What 3D file formats can ITK import and export?
+
+The following table identifies the file formats that ITK can read/write
+via a factory mechanism based on the file extension
+([`itk::ImageIOFactory`](https://itk.org/Doxygen/html/classitk_1_1ImageIOFactory.html)). Some proprietary files format are only imported.
+More information on [ITK File Formats](https://itk.org/Wiki/ITK_File_Formats) is
+available, including data types supported.
+
+A very nice tool for viewing all of these images is [ITKApps/ImageViewer](https://github.com/InsightSoftwareConsortium/ITKApps/tree/master/ImageViewer).
```

Comment:
All mentions to the `ImageViewer` should be maybe changed to the ITK/VTK WebJS viewer?

Reply:
Done in 894e5e6.